quinn-proto/src/connection/streams/mod.rs

```diff
@@ -318,6 +332,7 @@ fn push_pending(pending: &mut BinaryHeap<PendingLevel>, id: StreamId, priority:
     });
 }
 
+#[derive(Debug)]
```

Comment:
Was this intentional?

Reply:
I likely added it while debugging the problem. Can either keep or remove

Reply:
I weakly favor feeding rustc less code for this sort of thing, but I don't care that much.

Reply:
I removed it now. If it's required by someone it's trivial to add back